cmd/app/main.go

```
@@ -0,0 +1,72 @@
+package main // Package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+
+	"gorest-api/internal/config"
+	"gorest-api/internal/handler"
+	"gorest-api/internal/logs"
+	"gorest-api/internal/repository"
+	"gorest-api/internal/service"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/logger"
+	"github.com/joho/godotenv"
+)
+
+// @title           Go REST API
+// @version         1.0
+// @description Projects REST API for Go.
+
+// @host localhost:8080
+// @BasePath /
+
+// @securityDefinitions.apikey ApiKeyAuth
+// @in header
+// @name Authorization
+func main() {
+	_ = logs.InitLogger()
```

You should not ignore an error here. Stop the app so you can figure out what is going on with your program. Also, if this fails, it will panic somewhere else in the code and it would be harder to track it down